### PR TITLE
Jdp230102 30 order products tab

### PR DESCRIPTION
**Pull request by gawrysiakg**
This is my proposition to solve this problem. If anyone has any other suggestions, please tell me. I'm not sure if it's supposed to look like this, but it works. Please check and review quickly, because we don't have time. 
If someone else wants to do this task, that's fine. I did this task in the middle of the night and maybe I forgot something, so please double check

### CODE REVIEW

File: src/main/java/com/kodilla/ecommercee/domain/OrderProduct.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    @ManyToOne
+    @JoinColumn(name = "ORDER_ID")
+    private Order order;
+    @ManyToOne
+    @JoinColumn(name = "PRODUCT_ID")
+    private Product product;
+
+    private BigDecimal price;
+
+    private Long qty=0L;
```
Comment (Feliscatus8) [inline src/main/java/com/kodilla/ecommercee/domain/OrderProduct.java:31]: I think the value 0L is unnecessary, it's going to be set in the constructor anyway, so i'd just do private Long qty;

**Review by Feliscatus8 — APPROVED** [review]
looks good

**Review by MMacina — APPROVED** [review]

**Review by viepovsky — APPROVED** [review]
If it works then seems to be ok

**Review by DawidNst — APPROVED** [review]
I think it's ok

**Pull request merged**